virttest/vt_resmgr/resources/storage/dir/dir_volume.py

```
@@ -0,0 +1,52 @@
+LOG = logging.getLogger("avocado." + __name__)
+
+
+class DirFileVolume(FileVolume):
+    """
+    The local filesystem file-based volume resource
+    """
+
+    def bind_backings(self, nodes):
+        # Note a local dir resource has one and only one binding.
+        if self.binding_backings:
+            raise RuntimeError(f"Cannot bind a bound dir volume {self.name}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)